fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -88,6 +97,88 @@ rocksdb::ReadOptions getReadOptions() {
 	return options;
 }
 
+ACTOR Future<Void> rocksDBMetricLogger(std::shared_ptr<rocksdb::Statistics> statistics, rocksdb::DB* db) {
+	state std::vector<std::tuple<const char*, uint32_t, uint64_t>> tickerStats = {
```

Comment:
Does it make sense to log metrics on SSTables for each level? Having high level compaction stats is useful here.

Or you think we have to different PR to direct RocksDB `LOG` file trace event?

Reply:
I think a separate PR makes sense as compaction stats and per-level stats will be much more complicated.